libraries/provider_install_package.rb

```diff
@@ -0,0 +1,65 @@
+class Chef
+  # Chef Provider for installing or removing Elasticsearch from package or source
+  class Provider::ElasticsearchInstallPackage < Provider
+    include Poise
+    include ElasticsearchCookbook::Helpers
+
+    if respond_to?(:provides)
```

Comment:
I did this to maintain chef 11 compatibility, unfortunately. I could remove the check if we drop chef 11 compat. I'll put it on one line :)